src/components/Layout.jsx

```diff
@@ -0,0 +1,9 @@
+// import React from 'react';
+
+// const Layout = ({children}) => {
+//   return (
+//     <main>{children}</main>
+//   )
+// }
+
+// export default Layout;
```

Comment:
Unused?

Reply:
Yep. I had the intent of eventually creating a nav with more to it than just My Cart, but I'll cross that bridge when I get there. Will delete for now for cleanliness' sake.